server/filestore.go

```diff
@@ -3412,37 +3487,42 @@ func (mb *msgBlock) cacheLookup(seq uint64) (*fileStoredMsg, error) {
 	}
 
 	// Parse from the raw buffer.
-	subj, hdr, msg, mseq, ts, err := msgFromBuf(buf, hh)
-	if err != nil {
+
+	//	subj, hdr, msg, mseq, ts, err := msgFromBuf(buf, sm, hh)
```

Comment:
Remove?

Reply:
Done in previous update. I saw it as I was edited filestore.go. Thx.